core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,39 @@
+const e2eSuperAdmin = require('../puppeteer-testing-utilities/blogPostAdminUtils.js');
+const testConstants = require('../puppeteer-testing-utilities/testConstants.js');
+
+const blogAdminUrl = testConstants.URLs.BlogAdmin;
+
+async function blogAdminUpdatingRolesAndTagsProperties() {
+  const superAdmin = await new e2eSuperAdmin();
+
+  await superAdmin.openBrowser();
+  await superAdmin.signUpNewUserWithUserNameAndEmail('blogAdm', 'blog_admin@example.com');
+  await superAdmin.logout();
+
+  await superAdmin.signUpNewUserWithUserNameAndEmail('blogPostEditor', 'blog_post_editor@example.com');
+  await superAdmin.logout();
+
+  await superAdmin.signUpNewUserWithUserNameAndEmail('superAdm', 'testadmin@example.com');
+  await superAdmin.assignBlogAdminRoleToUserWithUserName('superAdm');
+  await superAdmin.expectUserToHaveBlogAdminRole();
+
+  await superAdmin.goto(blogAdminUrl);
+  await superAdmin.assignUserAsRoleFromRoleDropdown('blogAdm', 'BLOG_ADMIN');
+  await superAdmin.expectUserGivenTheRoleFromRoleDropdown('blogAdm', 'BLOG_ADMIN');
+  await superAdmin.assignUserAsRoleFromRoleDropdown('blogPostEditor', 'BLOG_POST_EDITOR');
+  await superAdmin.expectUserGivenTheRoleFromRoleDropdown('blogPostEditor', 'BLOG_POST_EDITOR');
+
+  await superAdmin.removeBlogEditorRoleByUsername('blogPostEditor');
+  await superAdmin.expectRemovedBlogEditorRoleByUsername('blogPostEditor');
```

Comment:
This doesn't make sense, the expectation shouldn't be about an action. It should be about a current state. So for example you would expect that the given user doesn't have a blogPostEditor role, not that the role is removed for that user -- the former is a "current state" and the latter is an "action".